src/components/ContactSalesForm/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react'
+import Button from '../../components/common/Button'
+import { ReactComponent as ChevronRightSvg } from '../../img/chevron-right.svg'
+import { ReactComponent as CheckSvg } from '../../img/check.svg'
+import { ReactComponent as CloseSvg } from '../../img/close.svg'
+import { FormState } from '../../types/FormState'
+import { sendEvent } from '../../utils/gtm'
+import classNames from 'classnames'
+import styles from './ContactSalesForm.module.scss'
+
+interface ContactSalesFormProps {
+  className?: string | string[]
+}
+export default function ContactSalesForm({ className }: ContactSalesFormProps) {
+  const submitEndpoint = process.env.FPJS_LEAD_URL
```

Comment:
We need to move it to env.ts, you also need to add GATSBY prefix in order to make it work